GoogleForCreators/web-stories-wp
# Redesign: Library header

## Context
Updates the Library header tabs and the media upload button.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
- Tabs have updated icons
- Text tab animation behavior has changed slightly -- the hovered state stays on the icon that was last hovered on.
- Upload button is using the primary button from the design system,.


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Verify that the looks and behavior of the library tabs is matching [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3408%3A84442)
2. Verify the Media Upload button is matching [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2241%3A2).

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6207
